### PR TITLE
PRA-132 Form for adding recipes manually

### PR DESCRIPTION
**Pull request by AnnaSolovykh**
## Requirements

 There is a form to add a recipe manually on route /manual-recipe

## Notes

Also navbar logo and bg color are added, some styling on Search AI input, flush variant of Inputs.

## Test Steps
 On  /manual-recipe you add a new recipe and get redirected to saved-recipes page.

## UI/UX Outcome

Screen Shot 2023-08-18 at 23 12 01

Screen Shot 2023-08-18 at 23 12 16

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Comment by AnnaSolovykh** [comment]
Sorry there is a kind of error - I have pulled the following Anna's PR with the Custom theme to adjust the theme to my code  - and now it looks like these changes are mine - but they not. My bad, sorry.

**Comment by AnnaSolovykh** [comment]
@rebekahcallkacz hi! Would you be so kind to take a look at my onChange types in react-select? I can't make out how to fix the types, tried different options.

**Review by AnnaPestova1 — APPROVED** [review]
Now we can add the recipe manually!!! Great!

File: src/components/EditRecipe/EditRecipe.tsx [hunk 1]
```diff
@@ -222,7 +222,7 @@ const EditRecipe = () => {
                     <option value="Side Dish">Side Dish</option>
                     <option value="Sandwich">Sandwich</option>
                     <option value="Picnic Ideas">Picnic Ideas</option>
-                    <option value="Smoothy">Smoothy</option>
+                    <option value="Smoothie">Smoothy</option>
```
Comment (Ekaterina-Bondareva) [inline src/components/EditRecipe/EditRecipe.tsx:225]: Could you please fixe Smoothy to Smoothie inside of <option> tag.

**Review by Ekaterina-Bondareva — APPROVED** [review]
LGTM! 
Could you please fixe Smoothy to Smoothie inside of tag. in EditRecipe.tsx line 225

**Comment by rebekahcallkacz** [comment]
> @rebekahcallkacz hi! Would you be so kind to take a look at my onChange types in react-select? I can't make out how to fix the types, tried different options.
> 
> UPDATE I have managed to fix the type errors but I am not sure if my ways are the best practice. It would be great if you could check anyway. Thank you!

I think that this is fine. `react-select` does a terrible job with types and creating more specific types is going to create a **lot** more work which probably isn't worth the effort. If you're interested, you can try out the generic types that `react-select` has (see [here](https://react-select.com/typescript)), but I think what you have is good!

**Pull request merged**